swap_meet/clothing.py

```diff
@@ -0,0 +1,52 @@
+
+# WAVE 5
+
+from .item import Item
+
+class Clothing(Item):
+
+    def __init__(self, category = "", condition=0, age = 0): 
```

Comment:
since Clothing class will always have "clothing" as its category, we can get rid of this parameter altogether!

```suggestion
    def __init__(self, condition=0, age = 0): 
```